sdk/tables/data-tables/src/utils/connectionString.ts

```diff
@@ -163,3 +166,17 @@ function getAccountNameFromUrl(url: string): string {
 
   return accountName;
 }
+
+/**
+ * Utility function that calculates the secondary URL for a table instance given the primary URL.
+ */
+export function getSecondaryUrlFromPrimarystri(primaryUrl: string): string {
```

Comment:
I'm a bit confused by the `stri` suffix?

Reply:
It is a typo :P